Library/Homebrew/test/dev-cmd/audit_spec.rb

```diff
@@ -1139,5 +1146,59 @@ class FooAT11 < Formula
         expect(fa.problems).to be_empty
       end
     end
+
+    describe "#audit_conflicts" do
+      before do
+        allow(File).to receive(:open).and_return("")
+      end
+
+      specify "it warns when conflicting with non-existing formula" do
+        foo = formula("foo") do
+          url "https://brew.sh/bar-1.0.tgz"
+
+          conflicts_with "bar"
+        end
+
+        fa = formula_auditor foo
```

Comment:
```suggestion
        fa = described_class.new foo
```

See my comment above